src/tactic/positivity.lean

```diff
@@ -297,18 +297,42 @@ private lemma div_nonneg_of_nonneg_of_pos [linear_ordered_field R] {a b : R} (ha
   0 ≤ a / b :=
 div_nonneg ha hb.le
 
+private lemma int_div_self_pos {a : ℤ} (ha : 0 < a) : 0 < a / a :=
+by { rw int.div_self ha.ne', exact zero_lt_one }
+
+private lemma int_div_nonneg {a b : ℤ} (ha : 0 ≤ a) (hb : 0 ≤ b) : 0 ≤ a / b := int.div_nonneg ha hb
```

Comment:
Seems like this is exactly defeq to `int.div_nonneg` and doesn't need to exist?

Reply:
Oh I thought you were duplicating all lemmas used to interface more cleanly with the rest of the library.